### PR TITLE
Create a listing card component and add a dark mode button

### PR DESCRIPTION
**Pull request by Vivek-ideates**
- created a listing card component
- created a dark mode button for the page
- added nodemon package ( command: npm run backend)

### CODE REVIEW

File: we-rent-app/src/components/Darkmode.tsx
Comment (pauravhp) [inline we-rent-app/src/components/Darkmode.tsx:1]: Change name of file to follow titlecase - DarkMode.tsx. Also components should be Components

File: we-rent-app/src/components/ListingCard.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                        </Button>
+                        <Button>
+                            <ThumbDownOffAltSharpIcon />
+                        </Button>
+                        <Button>
+                            <BookmarkBorderSharpIcon />
+                        </Button>
+                        <Typography
+                            variant="h6"
+                            color="textPrimary"
```
Comment (pauravhp) [inline we-rent-app/src/components/ListingCard.tsx:95]: I think it looks good for now, we can revamp it later on. But could you make the number display in white text for now?

File: we-rent-app/src/App.tsx [hunk 2]
```diff
@@ -31,26 +33,25 @@ const App = () => {
...
+		<>
+			<Container>
+				<MyBar sx={{ p: 1, textAlign: "left" }}>
+					<Typography variant="h4" sx={{ p: 1, textAlign: "left" }}>
+						{" "}
+						WeRent
 					</Typography>
-				))}
-		</Container>
+					<div style={{ position: 'absolute', top: '10px', right: '10px' }}>
```
Comment (pauravhp) [inline we-rent-app/src/App.tsx]: Is it possible to not have the pixel values and instead use non-CSS methods? If it takes too much time, leave it and we can figure for later

File: we-rent-app/package.json [hunk 3]
```diff
@@ -27,7 +28,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "backend": "node server/server.js"
```
Comment (pauravhp) [inline we-rent-app/package.json:35]: What's this?

Comment (pauravhp) [reply we-rent-app/package.json:35]: Nvm, saw the command you put

File: we-rent-app/src/components/Darkmode.tsx
Comment (Vivek-ideates) [reply we-rent-app/src/components/Darkmode.tsx:1]: done

**Pull request merged**